.github/workflows/build.yml

```
@@ -32,24 +34,28 @@ jobs:
         with:
           path: ~/conda_pkgs_dir
           key: ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{ hashFiles('environment_build.yml', 'requirements.txt') }}
+
       - name: Setup Miniconda
-        # https://github.com/conda-incubator/setup-miniconda
-        uses: conda-incubator/setup-miniconda@v2.0.1
+        uses: conda-incubator/setup-miniconda@v3.0.3
         with:
-          python-version: 3.7
-          use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
-          environment-file: environment_build.yml
+          miniforge-version: latest
           condarc-file: ${{ matrix.condarc }}
+          python-version: ${{ matrix.pyver }}
+          environment-file: environment_build.yml
           activate-environment: sleap_ci
+          conda-solver: "libmamba"
+          # use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
+
       - name: Print environment info
         shell: bash -l {0}
         run: |
           which python
           conda info
+          conda list
 
-      # Build pip wheel (Ubuntu)
-      - name: Build pip wheel (Ubuntu)
-        if: matrix.os == 'ubuntu-22.04'
+      # Build pip wheel (Not Windows)
+      - name: Build pip wheel (Not Windows)
+        if: matrix.os != 'windows-2022'
```

Extra work since we only upload the ubuntu wheel right now - maybe revert?
```suggestion
      # Build pip wheel (Ubuntu)
      - name: Build pip wheel (Ubuntu)
        if: matrix.os == 'ubuntu-22.04'
```

---

This is how it is in `build_manual.yml`. Do you think I should change it in both places?